web/src/list_widget.js

```diff
@@ -495,3 +487,5 @@ export function handle_sort($th, list) {
     // and not if it is undefined.
     list.sort(sort_type, prop_name);
 }
+
+export const default_get_item = (item) => item;
```

Comment:
I can merge this as written, but I'm not convinced we wouldn't be happier with the default being set inside the `ListWidget` constructor if not passed; that seems a lot cleaner than making 20 files all ask for the identity function...

Reply:
Definitely, it's just a hack for now based on the conversation we had here - https://chat.zulip.org/#narrow/stream/6-frontend/topic/list_widget.20typescript.20migration/near/1558934.

But the future plan here is that all these list_widget instances that use the default identity function should instead be using ids so that they will be passing actual meaningful `get_item` callbacks.

Reply:
OK